orga/tests/unit/models/prescriber_test.js

```diff
@@ -5,6 +5,37 @@ import { run } from '@ember/runloop';
 module('Unit | Model | prescriber', function (hooks) {
   setupTest(hooks);
 
+  module('#hasCurrentOrganizationWithGARAsIdentityProvider', function () {
+    test('it should return false if the current organization has not GAR as identity provider', function (assert) {
+      // given
+      const store = this.owner.lookup('service:store');
+      const organization = store.createRecord('organization', { name: 'Willow school' });
+      const userOrgaSettings = run(() => store.createRecord('userOrgaSetting', { organization }));
+      const membership = run(() => store.createRecord('membership', { organizationRole: 'MEMBER', organization }));
+      const memberships = [membership];
+      const model = run(() => store.createRecord('prescriber', { memberships, userOrgaSettings }));
```

Comment:
A la lecture, j'ai eu du mal à comprendre l'usage du run dans ce test, du coup j'ai lancé sans et ça passe :) du coup je propose : 
```suggestion
     const store = this.owner.lookup('service:store');
      const organization = store.createRecord('organization', { name: 'Willow school' });
      const userOrgaSettings = store.createRecord('userOrgaSetting', { organization });
      const model = store.createRecord('prescriber', { userOrgaSettings });
```

(je veux bien savoir ce que permet le run() du coup ^^)

Reply:
Faut demander à la team prescription pour le run ^^ J'ai pas voulu modifier leurs standards.
Je corrige.

Reply:
@AndreiaPena  C'est dans la doc officielle des tests qu'on voit que l'utilisation du run est donnée en exemple  https://db-osc-fr1.scalingo.com/dashboard/631b407e4e1ec31686e859fd/overview